packages/dbos-cloud/cli.ts

```diff
@@ -395,14 +395,16 @@ workflowCommands
   .description('List workflows from your application')
   .argument("[string]", "application name (Default: name from package.json)")
   .option('-l, --limit <number>', 'Limit the results returned', "10")
-  .option('-u, --user <string>', 'Retrieve workflows run by this user')
+  .option('-u, --workflowUUIDs <uuid...>', 'Retrieve specific UUIDs')
+  .option('-U, --user <string>', 'Retrieve workflows run by this user')
```

Comment:
This is a breaking change. Why `-u`and not `-w`?

Reply:
Not a breaking change because this has never been released :)